pyproject.toml

```diff
@@ -58,7 +58,7 @@ install-autogen = """curl -fsSL --output /tmp/autogen.zip "https://github.com/mb
   mv /tmp/autogen/autogen-*/* /tmp/autogen && rm -rf /tmp/autogen/autogen-* && 
   sudo chmod a+rx /tmp/autogen/autogen.sh"""
 # TODO: install in Mac/Windows
-install-yapf = """sudo apt-get install -y yapf3 python3-yapf || echo 'yapf could not be installed'"""
+install-yapf = """pip3 install yapf==0.40.2 --break-system-packages || echo 'yapf could not be installed'"""
```

Comment:
re: `--break-system-packages` i think *we're* ok with this ("we" as in kaggle devs), but i could see someone else wanting to contribute to this project and not expecting to have this behavior.

we may want to look at setting up a python environment, or perhaps even better, a dev docker container for the Kaggle API so this would be contained to that. (and if we already have that, apologies for the noise)

Reply:
Ah, good point. I need to look into this further.

There is a Docker container that's used for distribution. I will see if I can launch it locally and do some testing.

Reply:
Agreed, this (creating a dev Docker container) is a long-standing request.  I'm not sure if it needs to be done immediately, but it seems to just keep biting us.